api/s2n.h

```diff
@@ -421,6 +421,97 @@ extern int s2n_connection_set_client_auth_type(struct s2n_connection *conn, s2n_
 S2N_API
 extern int s2n_connection_get_client_cert_chain(struct s2n_connection *conn, uint8_t **der_cert_chain_out, uint32_t *cert_chain_len);
 
+/**
+ * Sets the initial number of session tickets to send after a >=TLS1.3 handshake.
+ *
+ * @param config A pointer to the s2n_config object.
+ * @param num The number of session tickets that will be sent.
+ */
+S2N_API
+extern int s2n_config_set_initial_ticket_count(struct s2n_config *config, uint8_t num);
+
+/**
+ * Increases the number of TLS1.3 session tickets to send after a >=TLS1.3 handshake.
+ *
+ * @param conn A pointer to the s2n_connection object.
```

Comment:
I'm really not a fan of this style. Why do we only use the type name for the first parameter, and not any other arguments? Why isn't the function signature sufficient?

Reply:
I changed this back.